src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/DocumentComponentTest.kt

```diff
@@ -229,7 +231,7 @@ class DocumentComponentTest : ComponentTestBase() {
     val recall = authenticatedClient.bookRecall(bookRecallRequest)
     val document = authenticatedClient.uploadDocument(
       recall.recallId,
-      AddDocumentRequest(PART_A_RECALL_REPORT, base64EncodedDocumentContents, fileName)
+      AddDocumentRequest(PART_A_RECALL_REPORT, base64EncodedDocumentContents, fileName, null)
```

Comment:
again, think the null is defaulted on this class, so no need to pass it unless you feel the test is more explicit this way